src/Python/chevahbs

```diff
@@ -79,22 +79,28 @@ chevahbs_configure() {
 
 chevahbs_compile() {
     execute "${MAKE_CMD[@]}"
-    # FIXME:12: Enable Python's own tests again.
-    # Don't use execute below, some tests might fail innocuously.
-    # execute "${MAKE_CMD[@]}" test
+
+    if [ "$OS" = "macos" ]; then
```

Comment:
can we have this as a separate command?

The command is "compile" , but this is not compile + test.

I think it's easier if we have a separate command.

And we can skip calling  the test  based on the GHA yaml files

Reply:
I've added dedicated a `chevahbs` function for testing.

This testing is now listed distinctly in GHA output, alongside building and installing (installing can also be quite verbose, e.g. for OpenSSL and Python).